platform.json

```diff
@@ -33,6 +37,11 @@
       "type": "framework",
       "optional": true,
       "version": "~1.232.0"
+    },
+    "framework-artik-sdk": {
+      "type": "framework",
+      "optional": true,
+      "version": "1.0.2"
```

Comment:
Do you want to be fixed with strict version? How about `~1.0.2` which means `1.0.x`?